team.html

```diff
@@ -520,6 +524,7 @@ <h3 id="core-package-contributors">Core Package Contributors<a class="paralink"
 			<li>Max Mahlke</li>
 			<li>Max Silbiger</li>
 			<li>Max Voronkov</li>
+			<li>Maximilian Linhoff</li>
 			<li>Maximilian Nöthe</li>
```

Comment:
I think for this, it is a name change. cc @maxnoe
```suggestion
			<li>Maximilian Linhoff</li>
```

Reply:
Yes, thanks!